app/settings.py

```diff
@@ -8,3 +8,7 @@
 os.environ['LOGS_DIR'] = LOGS_DIR  # Set default if not set
 LOGGING_CFG = os.getenv('LOGGING_CFG', 'logging-cfg-local.yml')
 TRAP_HTTP_EXCEPTIONS = True
+
+# Definition of the allowed domains for CORS implementation
+ALLOWED_DOMAINS_STRING = os.getenv('ALLOWED_DOMAINS', r'http[s]?://localhost')
```

Comment:
I would either use `'.*'` as default or no default at all.

Reply:
yep good point, I've removed the default value (it comes through the `.env.local` file anyway)